id-checklist

```diff
@@ -49,20 +49,25 @@ extension of this lifetime is possible; see Section X below.
 After a document becomes an RFC, it will be marked as replaced by that RFC
 in the I-D repository.
 
-[MSK: This paragraph and the next probably need revision.]
-[BJK: Yes!]
-
-I-Ds are generally in the format of an RFC, although they may be rough drafts
```

Comment:
"format" here was overloaded for both the plain-text bit and the layout; I dropped it down to just being the layout/structure since the format of an RFC is now multi-valued.